helpers/schemas.js

```diff
@@ -0,0 +1,12 @@
+import { object, string } from "yup";
+
+export const registerSchema = object().shape({
+  username: string()
+    .required("Username is required")
+    .min(3),
+  password: string()
+    .required('No password provided.')
+    .min(6, 'Password is too short - should be 6 chars minimum.')
+    .matches(/[a-zA-Z]/, 'Password can only contain Latin letters.')
```

Comment:
Not a blocker, but good password requires you to use upper letters, special symbols and numbers. Password that contains only latin letters will be brutforced in 10 mins